superduperdb/base/build.py

```diff
@@ -158,7 +158,9 @@ def build_compute(compute):
     if compute.split('://')[0] == 'ray':
         return RayComputeBackend(compute)
 
-    return LocalComputeBackend()
+    compute = LocalComputeBackend()
```

Comment:
The only way to reach this state is to give an invalid URL -- something that is not None, and is not one of the valid conditions.

In this case, you should return an exception.